### PR TITLE
Inject annotation plugin to hosted html files

### PR DESCRIPTION
**Pull request by tuliren**
Inject `@annotatejs/web` into hosted html files. This plugin currently only support text highlighting locally. But it will eventually evolve into a full Vercel-style feedback system. Check out the [website](https://www.annotatejs.dev) and npm package [`@annotatejs/web`](https://www.npmjs.com/package/@annotatejs/web) for details.

### CODE REVIEW

File: action/src/plugins/staticHosting/helpers.ts [hunk 1]
```diff
@@ -168,8 +169,31 @@ export const uploadFileWithSignedUrl = async (
   }
 };
 
+const injectAnnotationPlugin = (localFilePath: string) => {
+  if (!localFilePath.toLowerCase().endsWith('.html')) {
```
Comment (jrhizor) [inline action/src/plugins/staticHosting/helpers.ts:173]: What if it's a directory that also has an index.html file in it?

**Review by jrhizor — APPROVED** [review]
LGTM (except for comment about handling directories)

File: action/src/plugins/staticHosting/helpers.ts (hunk 1 above)
Comment (tuliren) [reply action/src/plugins/staticHosting/helpers.ts:173]: This helper only process files. It is called in two places: one for file and one for directories. When it is a directory, the caller will iterate through each file.

Comment (tuliren) [reply action/src/plugins/staticHosting/helpers.ts:173]: The non-single page test coverage report is a directory with a group of html files.

Comment (jrhizor) [reply action/src/plugins/staticHosting/helpers.ts:173]: Cool.

**Pull request merged**